docs/man.rst

```diff
@@ -176,9 +176,10 @@ index>`` entry will also be present.
     pick 735609912343 Summary of second commit
     index 672841329981 <git index>
 
-These commits may be re-ordered to change the order they appear in history.
-In addition, the ``pick`` and ``index`` commands may be replaced to modify
-their behaviour.
+These commits may be re-ordered to change the order they appear in history. If
+present, then ``index`` commands must be at the bottom of the list, i.e. they
+can not be followed by non-index commands. In addition, the ``pick`` and
+``index`` commands may be replaced to modify their behaviour.
```

Comment:
Super nit-picky (sorry!), but I'd prefer to make the index caveat be the last line.